ansible/roles/tasks-systemd/tasks/main.yml

```diff
@@ -63,18 +63,29 @@
           '--memory=24g',
           '--pids-limit=16384',
           '--shm-size=1024m',
+
           # qcow overlays on tmpfs
           '--tmpfs=/tmp:size=14g',
           '--env=TEST_OVERLAY_DIR=/tmp',
+
+          # image cache
+          '--env=COCKPIT_IMAGES_DATA_DIR=/cache/images',
+          '--volume=/var/cache/cockpit-tasks/images:/cache/images:rw',
+
           # local image stores
           '--env=COCKPIT_IMAGE_STORES_FILE=/config/image-stores',
           '--volume=/var/cache/cockpit-tasks/image-stores:/config/image-stores:ro',
+
           # generic secrets
           '--volume=/var/lib/cockpit-secrets/tasks/npm-registry.crt:/run/secrets/tasks/npm-registry.crt:ro',
+
           # various configuration
           '--volume=/etc/npmrc:/etc/npmrc:ro',
-          '--volume=/var/cache/cockpit-tasks/images:/cache/images:rw',
           '--env=TEST_JOBS={{ TEST_JOBS | default(8) }}',
+          '--env=GIT_COMMITTER_NAME=Cockpituous',
```

Comment:
Ditto about `run-local.sh`

Reply:
Right -- doesn't do anything, but I added it for completeness. Same for the CentOS CI deployment.